test/test.oauth2.ts

```diff
@@ -855,7 +854,7 @@ describe('OAuth2 client', () => {
     const auth = new GoogleAuth();
     const oauth2client =
         new auth.OAuth2(CLIENT_ID, CLIENT_SECRET, REDIRECT_URI);
-    oauth2client.request(({} as request.OptionsWithUri), (err, result) => {
+    oauth2client.request(({} as AxiosRequestConfig), (err, result) => {
```

Comment:
All properties in `AxiosRequestConfig` are optional, so the cast should not be needed.

Reply:
fixed here and a few other places